routes/user/index.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express')
+const route = express.Router()
+const ControllerUser = require('../../controllers/user')
+const {authentication} = require('../../middleware/isAuthentication')
+const {authorization } = require('../../middleware/isAuthorization')
+
+route.use('/',authentication)
+route.get('/',authorization,ControllerUser.getAllUser)
```

Comment:
Spasinya jangan pelit pelit

```suggestion
route.get('/' ,authorization, ControllerUser.getAllUser)
```